go.sum

```diff
@@ -1,9 +1,11 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7gZCb22OMCxBHrMx7a5I7Hp++hsVxbQ4BYO7hU=
```

Comment:
the custom domains are freaking me out. wonder which dependency is pulling it it.

Reply:
Maybe not the only path, but:

```console
$ go mod graph | grep dmitri.shuralyov.com/gpu/mtl
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 dmitri.shuralyov.com/gpu/mtl@v0.0.0-20190408044501-666a987793e9

$ go mod graph | grep golang.org/x/exp
github.com/grpc-ecosystem/go-grpc-middleware@v1.2.2 golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 dmitri.shuralyov.com/gpu/mtl@v0.0.0-20190408044501-666a987793e9
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 github.com/BurntSushi/xgb@v0.0.0-20160522181843-27f122750802
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20200222043503-6f7a984d4dc4
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 golang.org/x/image@v0.0.0-20190802002840-cff245a6509b
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 golang.org/x/mobile@v0.0.0-20190719004257-d2bd2a29d028
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 golang.org/x/mod@v0.1.1-0.20191107180719-034126e5016b
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 golang.org/x/sync@v0.0.0-20190423024810-112230192c58
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 golang.org/x/sys@v0.0.0-20191001151750-bb3f8db39f24
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 golang.org/x/tools@v0.0.0-20200207183749-b753a1ba74fa
golang.org/x/exp@v0.0.0-20200331195152-e8c3332aa8e5 golang.org/x/xerrors@v0.0.0-20191011141410-1b5146add898
golang.org/x/mobile@v0.0.0-20190719004257-d2bd2a29d028 golang.org/x/exp@v0.0.0-20190306152737-a1d7652674e8
google.golang.org/genproto@v0.0.0-20190819201941-24fa4b261c55 golang.org/x/exp@v0.0.0-20190121172915-509febef88a4
google.golang.org/genproto@v0.0.0-20190927181202-20e1ac93f88c golang.org/x/exp@v0.0.0-20190121172915-509febef88a4
google.golang.org/genproto@v0.0.0-20190307195333-5fe7a883aa19 golang.org/x/exp@v0.0.0-20190121172915-509febef88a4
google.golang.org/genproto@v0.0.0-20190530194941-fb225487d101 golang.org/x/exp@v0.0.0-20190121172915-509febef88a4
google.golang.org/genproto@v0.0.0-20190425155659-357c62f0e4bb golang.org/x/exp@v0.0.0-20190121172915-509febef88a4

$ go mod why github.com/grpc-ecosystem/go-grpc-middleware
warning: ignoring symlink /home/jpkroehling/Projects/src/github.com/jaegertracing/jaeger/jaeger-ui/node_modules/@jaegertracing/plexus
warning: ignoring symlink /home/jpkroehling/Projects/src/github.com/jaegertracing/jaeger/jaeger-ui/node_modules/jaeger-ui
# github.com/grpc-ecosystem/go-grpc-middleware
github.com/jaegertracing/jaeger/cmd/flags
github.com/grpc-ecosystem/go-grpc-middleware/logging/zap
github.com/grpc-ecosystem/go-grpc-middleware

```